code/modules/projectiles/projectile/magic.dm

```diff
@@ -580,3 +580,10 @@
 	damage_type = BURN
 	damage = 2
 	antimagic_charge_cost = 0 // since the cards gets spammed like a shotgun
+
+/obj/projectile/beam/shrink/magic
```

Comment:
Because this is not subtyped off of `/obj/projectile/magic`, this will not be blocked by antimagic. Which is not ideal?

Reply:
Hmmm. Yeah I was wondering if that would matter. Might just swap the whole thing over to the magic subtype and abductors can just have deep lore that they're secretly using magic or something

Reply:
That would work for me